### PR TITLE
Add parameter overrides to opener files using json

### PR DESCRIPTION
**Pull request by Hikari-Haru**
Added the ability to load .json files which include temp, top p, frequency and presence penalties. Also fixed some path traversal possibilities with opener files. Reordered the parts of `/gpt converse` some too. A review and sanity check that the path traversal isn't possible anymore would be appreciated

In reference to discussion started by @uberdude420

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -80,6 +80,8 @@ These commands are grouped, so each group has a prefix but you can easily tab co
 
 - Custom openers need to be placed as a .txt file in the `openers` directory, in the same directory as `gpt3discord.py`
 
+- Can use .json files in the `{"text": your prompt, "temp":0, "top_p":0,"frequency_penalty":0,"presence_penalty":0}` format to include permanent overrides
```
Comment (Kav-K) [inline README.md:83]: Could you fix conflicts and maybe add a bit more info on how to use this? maybe with an example or tell them to refer to the sample .json file?

**Review by Kav-K — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Hikari-Haru) [reply README.md:83]: Added some to the readme

**Review by Kav-K — APPROVED** [review]

**Comment by Hikari-Haru** [comment]
Wait a min on that merge actually

**Comment by Kav-K** [comment]
kk

**Comment by Hikari-Haru** [comment]
It added the opener_file part twice after the merge lol, should be fine now

**Review by Kav-K — APPROVED** [review]

**Pull request merged**